docs/plugins.md

```diff
@@ -90,6 +90,28 @@ $ probot run -a APP_ID -P private-key.pem ./index.js
 Listening on http://localhost:3000
 ```
 
+## HTTP Routes
+
+`robot.router` is an [express](http://expressjs.com/) app that will serve HTTP requests.
+
+```js
+const express = require('express');
+
+module.exports = robot => {
+  // Use any express middleware
+  robot.router.use(express.static(__dirname + '/public'));
+
+  //
+  robot.router.get('/hello-world', (req, res) => {
+    res.end('Hello World');
+  });
+};
```

Comment:
I think this should be something like: (based on how the code looks in the test cases)
```
const app = robot.route('prefix');
app.use(express.static(__dirname + '/public'));
app.get('/hello-world', ...
```
Would also be dope to add a comment explaining the need to namespace your plugin. Ex:

Calling `robot.route('someprefix')` will return an express router that you can use to allow your probot plugin to expose HTTP endpoints. 

**Note**: We *strongly encourage* you to use the name of your package as the prefix so none of your routes conflict with other plugins. Ex: If [`probot/owners`](https://github.com/probot/probot) wanted to expose an endpoint, the plugin would call `robot.route('owners')` to prefix all endpoints with `owners`.